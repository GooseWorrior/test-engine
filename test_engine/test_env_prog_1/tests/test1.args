this is the argument file for test 1
