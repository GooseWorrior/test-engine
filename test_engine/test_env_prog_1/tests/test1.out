this is the output file for test1
